doctrine/doctrine-bundle/1.12/config/packages/doctrine.yaml

```diff
@@ -0,0 +1,23 @@
+doctrine:
+    dbal:
+        url: '%env(resolve:DATABASE_URL)%'
+
+        # IMPORTANT: You MUST configure your db driver and server version,
+        # either here or in the DATABASE_URL env var (see .env file)
+        #driver: 'mysql'
+        #server_version: '5.7'
+
+        # Only needed for MySQL (ignored otherwise)
+        charset: utf8mb4
+        default_table_options:
+            collate: utf8mb4_unicode_ci
+    orm:
+        naming_strategy: doctrine.orm.naming_strategy.underscore_number_aware
+        auto_mapping: true
+        mappings:
+            App:
+                is_bundle: false
+                type: annotation
+                dir: '%kernel.project_dir%/src/Entity'
+                prefix: 'App\Entity'
+                alias: App
```

Comment:
Weren't aliases supposed to be removed? Or I might have forgotten about the discussion 🤔 😕

Reply:
They might be removed in ORM 3.0, but they are still part of the config, so I wouldn't change that part right now.